scripts/scripts/select_fill_in_questions.py

```
@@ -0,0 +1,191 @@
+    if output_path == "-":
+        return sys.stdout
+    try:
+        return open(output_path, "w", encoding="utf-8")
+    except Exception as e:
+        print(f"Error opening output file '{output_path}': {e}", file=sys.stderr)
+        sys.exit(1)
+
+
+def generate_prompt(entry: Dict[str, Any], language: str) -> str:
+    theme = entry.get("entry", {}).get("theme", "")
+    level = entry.get("entry", {}).get("level", "")
```

Ruff (0.8.2)

Local variable `level` is assigned to but never used

Remove assignment to unused variable `level`

(F841)